src/components/Process/CSP/basics.tsx

```diff
@@ -44,8 +46,17 @@ export const useTwoFactorAuth = <T extends number>(process: PublishedElection, s
         body: JSON.stringify(payload),
       })
       if (!response.ok) {
-        throw new Error((await response.json()).error)
+        const errorMessage = (await response.json()).error
+
+        const translatedMessage = /participant not found in the census/i.test(errorMessage)
```

Comment:
I think it'll be better if we check for the error number, not the error text. Don't we have a specific error number for this case?